packages/utils/src/validation/validatorFunctions.js

```diff
@@ -150,9 +158,9 @@ export const constructRecordExistsWithId = (modelOrDatabase, recordType) => asyn
   }
 };
 
-export const constructIsEmptyOr = validatorFunction => value => {
+export const constructIsEmptyOr = validatorFunction => (value, ...args) => {
```

Comment:
This is clever! I suggest making it slightly more readable by making it explicit:

```suggestion
export const constructIsEmptyOr = validatorFunction => (value, object, key) => {
```

and similarly in line 163

Reply:
This is more readable! I didn't realize validator functions had a common signature.

Reply:
It may not be obvious, but their function signature is a result of how they are called in `ObjectValidator.runValidators()`